website/docs/artifacts.md

```diff
@@ -287,3 +287,32 @@ Note that headers are not installed within a subdirectory of `/usr/include/`
 with the name of the package. They are installed directly into `/usr/include/`.
 For instance, for the above examples, the headers would be installed to
 `/usr/include/my_header.h` and `/usr/include/my_headers/` respectively.
+
+### Users
+
+Users allow you to specify a list of users to be created when installing the
+package.
+In most cases this will require a shell to be available on the target system.
```

Comment:
When would this not require a shell to be available on the target system?

Reply:
It's just up to the underlying implementation.
On debian we could potentially get away with no shell... I think... under certain circumstances if there are no other postinst scripts that are needed and we provided a static binary in the package to deal with this... as an example.

Reply:
Ah got it, that makes sense! Thanks for clarifying!